Header              header      # Header containing timestamp etc. of this message
TrackedObject[]     tracks      # All objects that are currently being tracked